### ISSUE TITLE
Switch to Keyboard.arrows for receiving arrow key inputs

### ISSUE THREAD

**Issue by specious** [kghandhi/pacman#2]
In #1 it has been noted that the arrow keys aren't working in Chrome (but do in Firefox).

Using this [Elm signal visualizer](http://yang-wei.github.io/elmflux/#/keyboardSignal), I observed that `Keyboard.arrows` is a reliable input for arrow key presses whereas `Keyboard.presses` isn't.

The inconsistent behavior of the `Keyboard.presses` signal with regards to the arrow keys is a known issue (see https://github.com/elm-lang/core/pull/463) which stems from differing interpretations of the `keypress` event by the browsers.

Switching to using `Keyboard.arrows` seems like the best way to solve this problem.

**Comment by asecular** [comment]
I think that could work, but we'd need to use a modified version of Keyboard.arrows since the normal version can't differentiate the sequences [left pressed, left unpressed] and [left pressed, right pressed]. Pacman should be moving left in the first case and right in the second case, or at least that's the current behavior. If we use an edited version of the code [here](https://github.com/elm-lang/core/blob/3.0.0/src/Keyboard.elm) where we replace `toXY` and `arrows` with something along the lines of

```
toUDLR : Directions -> Set.Set KeyCode -> { u : Bool, d : Bool, l : Bool, r : Bool }
toUDLR {up,down,left,right} keyCodes =
  let is keyCode =
        Set.member keyCode keyCodes
  in
      { u = is up
      , d = is down
      , l = is left
      , r = is right
      }

arrows : Signal { u : Bool, d : Bool, l : Bool, r : Bool }
arrows =
  dropMap (toUDLR { up = 38, down = 40, left = 37, right = 39 }) keysDown
```

I think we can get around that issue as long as we keep track of the current {u,d,l,r} tuple somewhere in the state so we can see what arrow key changed.

**Comment by kghandhi** [comment]
I just got this working, its a little ugly but I need to brush up on my elm to make it prettier. Ill work on cleaning up some of this code in the next few weeks and figuring out how to update the gh-pages link so that it displays the code thats currently on master.

**Comment by specious** [comment]
[elm-format](https://github.com/avh4/elm-format) tool can help keep the code clean.  I think this issue has been fixed in 7fe2be26a7a5c8e45a86707b70540cf37fe08074.  Great work, @kghandhi :)
